aioftp/client.py

```diff
@@ -620,7 +640,13 @@ class Client(BaseClient):
         :param path: new directory, goes «up» if omitted
         :type path: :py:class:`str` or :py:class:`pathlib.PurePosixPath`
         """
-        if path in ("..", pathlib.PurePosixPath("..")):
+        self.cwd = None
+
+        if type(path) is str:
+
+            path = pathlib.PurePosixPath(path)
```

Comment:
* For type check `isinstance` is better, since it is the only purpose of `isinstance`.
* You can wrap any input (`str` or `PurePosixPath`) with `PurePosixPath`, so no check is actually needed.